src/pages/AdditionalInformation/TermsOfUse.jsx

```diff
@@ -17,16 +17,15 @@ const TermsOfUse = () => {
             >
               Često postavljana pitanja
             </NavLink>
-            <br />
-
+            <br /> <br />
             <NavLink
               to='/privacynotices'
               className='text-for-terms-use'
               activeClassName='active'
             >
               Obaveštenje o privatnosti
```

Comment:
Why do we have same menu in 3 different screens. You should use different component just for that(menu) or make some bigger part of page reusable

Reply:
Deleted

Reply:
In another PR, or?